### ISSUE TITLE
POWER OFF dGPU

### ISSUE THREAD

**Issue by bouhka** [dkoluris/ZEPHYRUS-M15-GU502-OpenCore#6]
Hi dkoluris, I have a legion with 10750h, i cannot turn off the power of gtx1660ti.
did you get turn off on your asus? 
In his readme comments that his fans are even hearing idle. 
I think this is a symptom that maybe dGPU is not off, it happens to me too.
I measured the total consumption(with energy meter) on windows in idle u is 15w, on mac is 50w
thank you

**Comment by koluris** [comment]
With the provided .aml files the dGPU is off on this model, but the fan can be heard in low spinning. Btw, CPU temperature is 30 - 35 degrees, instead when the dGPU wasn't off the temperature would hit 55 - 60 degrees.

**Comment by bouhka** [comment]
I have studied your ssdt-dgpuoff.aml, it is different from the one provided by dortania.  
what you changed in ssdt-dgpuoff.aml was to be able to turn off dgpu? was the one provided by dortania working for you? . my cpu is set to consumption 3w to 5w in idle, u temperature at 40. how much does the battery last? thank you

**Comment by koluris** [comment]
To be honest I don't remember where I got this one... what keeps the dGPU off is the other .aml PTSWAK and some patches inside config.plist. Battery life is pretty generous, it could last from 5 - 8 hours depending on the usage.

**Comment by bouhka** [comment]
wow 5 to 8 hours? that's a dream for me right now. It was spoken in forums that it was a legion 10 generation problem, but from what i observe the general dortania method does not work for asus because there are variations in its acpi as in others . Thank you for your information. I will continue investigating.

**Comment by koluris** [comment]
Yeah, when i just use it lightly surfing the web etc (no videos, music whatsoever) it lasts up to 8 hours... If I start compiling stuff with Xcode, watching videos and listening to music it drops down significantly. Also, keep in mind that I have a very conservative CPU PM profile with CPUFriend, which keeps the processor speed as low as 800MHz on Idle.

**Comment by koluris** [comment]
![Στιγμιότυπο 2021-03-03, 6 32 49 μμ](https://user-images.githubusercontent.com/31432553/109839462-cd168d80-7c4f-11eb-9b12-ed18eb1d7d5e.png)

Here's an hour and fifteen minutes of idle/light usage operation for reference.

**Comment by bouhka** [comment]
it is expected on a mac, if everything works well. In my previous hackintosh i multiplied 2.5 times the life of the battery compared to windows. In my current legion it does not arrive for 2 hours.
Its battery according to specifications is 76w-h, mine is 80 (17 "), we should have a similar duration, however my cpu in idel is 40-43 degrees.
